NetCord/Rest/RestGuild.cs

```diff
@@ -102,4 +62,274 @@ int GetHighestRolePosition(PartialGuildUser user)
+    /// </summary>
+    public ulong? ApplicationId => _jsonModel.ApplicationId;
+
+    /// <summary>
+    /// The ID of the channel where <see cref="RestGuild"/> notices such as welcome messages and boost events are posted.
+    /// </summary>
+    public ulong? SystemChannelId => _jsonModel.SystemChannelId;
+
+    /// <summary>
+    /// Represents the <see cref="RestGuild"/>'s current system channels settings.
+    /// </summary>
+    public SystemChannelFlags SystemChannelFlags => _jsonModel.SystemChannelFlags;
+
+    /// <summary>
+    /// The ID of the channel where community guilds can display their rules and/or guidelines.
+    /// </summary>
+    public ulong? RulesChannelId => _jsonModel.RulesChannelId;
+
+    /// <summary>
+    /// The maximum number of <see cref="Presence"/>s for the <see cref="RestGuild"/>. Always <see langword="null"/> with the exception of the largest guilds.
+    /// </summary>
+    public int? MaxPresences => _jsonModel.MaxPresences;
+
+    /// <summary>
+    /// The maximum number of <see cref="GuildUser"/>s for the <see cref="RestGuild"/>.
+    /// </summary>
+    public int? MaxUsers => _jsonModel.MaxUsers;
+
+    /// <summary>
+    /// The <see cref="RestGuild"/>'s vanity invite URL code.
+    /// </summary>
+    public string? VanityUrlCode => _jsonModel.VanityUrlCode;
+
+    /// <summary>
+    /// The <see cref="RestGuild"/>'s description, shown in the 'Discovery' tab.
+    /// </summary>
+    public string? Description => _jsonModel.Description;
+
+    /// <summary>
+    /// Whether the <see cref="RestGuild"/> has a set banner.
+    /// </summary>
+    public bool HasBanner => BannerHash is not null;
+
+    /// <summary>
+    /// The <see cref="RestGuild"/>'s banner hash.
+    /// </summary>
+    public string? BannerHash => _jsonModel.BannerHash;
+
+    /// <summary>
+    /// Gets the <see cref="ImageUrl"/> of the <see cref="RestGuild"/>'s banner.
+    /// </summary>
+    /// <param name="format">The format of the returned <see cref="ImageUrl"/>. Defaults to <see cref="ImageFormat.Png"/> (or <see cref="ImageFormat.Gif"/> for animated icons).</param>
+    /// <returns>An <see cref="ImageUrl"/> pointing to the guild's banner. If the guild does not have one set, returns <see langword="null"/>.</returns>
+    public ImageUrl? GetBannerUrl(ImageFormat? format = null) => BannerHash is string hash ? ImageUrl.GuildBanner(Id, hash, format) : null;
+
+    /// <summary>
+    /// The <see cref="RestGuild"/>'s current server boost level.
+    /// </summary>
+    public int PremiumTier => _jsonModel.PremiumTier;
+
+    /// <summary>
+    /// The number of boosts the <see cref="RestGuild"/> currently has.
+    /// </summary>
+    public int? PremiumSubscriptionCount => _jsonModel.PremiumSubscriptionCount;
+
+    /// <summary>
+    /// The preferred locale of a community <see cref="RestGuild"/>, used for the 'Discovery' tab and in notices from Discord, also sent in interactions. Defaults to "en-US".
```

Comment:
I think we should only use single or double quotes. Not to mix them. Also what about using `c` tag instead?

Reply:
Double quotes should only be used for literal quotes, single quotes are just for clarifying something. Changed the emphasis around `en-US` to be a `<c>` tag though.